### PR TITLE
Adding note on usage of Cluster Dispatcher #20775

### PR DESCRIPTION
**Pull request by hhimanshu**
- Add note as specified by @ktoso in https://github.com/akka/akka/issues/20775

### CODE REVIEW

**Comment by johanandren** [comment]
LGTM

File: akka-docs/rst/java/cluster-usage.rst [hunk 1]
```diff
@@ -733,3 +733,13 @@ For this purpose you can define a separate dispatcher to be used for the cluster
...
+
+.. note::
+    Normally it should not be necessary to configure a separate dispatcher for the Cluster.
+    The default-dispatcher should be sufficient for performing the Cluster tasks, i.e. ``akka.cluster.use-dispatcher``
+    should not be changed. If you have Cluster related problems when using the default-dispatcher that is typically an
+    indication that you are running blocking or CPU intensive actors/tasks on the default-dispatcher.
+    Use dedicated dispatchers for such actors/tasks instead of running them on the default-dispatcher,
+    because that may starve system internal tasks.
+    Related config properties: ``[akka.cluster.use-dispatcher = akka.cluster.cluster-dispatcher]``.
+    Corresponding default values: ``[akka.cluster.use-dispatcher = ]``
```
Comment (ktoso) [inline akka-docs/rst/java/cluster-usage.rst]: The `[` is a bit weird, could you remove it?

**Comment by ktoso** [comment]
LGTM after `[` removed, looks weird

**Comment by hhimanshu** [comment]
@ktoso , I am assuming you meant both `[` and `]` in 

```
 Related config properties: ``akka.cluster.use-dispatcher = akka.cluster.cluster-dispatcher``.
 Corresponding default values: ``akka.cluster.use-dispatcher = ``
```

Right? Please let me know. Meanwhile, I am removed them locally and building the docs again, waiting for your heads up
Thanks

**Comment by hhimanshu** [comment]
@johanandren, @ktoso, all well?

**Comment by ktoso** [comment]
Yeah, that's what I meant. Thanks! LGTM

**Comment by hhimanshu** [comment]
awesome, glad that I contributed to the project which I really like to work with :)

**Comment by ktoso** [comment]
Every contrib counts, thanks again!

**Pull request merged**